### ISSUE TITLE
Fresh Install Plasma - Flatpak is missing the flathub repository (T10122)

### ISSUE THREAD

**Issue by celticmagic** [getsolus/packages#109]
<div dir="rtl"><b>John&nbsp;Smith</b>&nbsp;(<em>&#35;rrfss</em>),&nbsp;2022-01-18&nbsp;00:41:22&nbsp;UTC</div><br/>

 
```
mattprice&#35;matt-pc ~ $ flatpak install flathub org.gabmus.hydrapaper
Looking for matches…
error: No remote refs found similar to ‘flathub’

```

I did a fresh install to play around with the plasma envionment, and while installing apps I found that flatpak doesn't work out of the box.
According to the [Flatpak page](https://flatpak.org/setup/Solus/) for Solus installation, the package `xdg-desktop-portal-gtk ` is supposed to be installed alongside it.
[Flatpaks GitHub for that package](https://github.com/flatpak/xdg-desktop-portal-gtk) mentions GNOME infrastructure, which I am obviously not using. eopkg confirmed it wasn't installed already. I decided to skip that and just run the next command to add the FlatHub repo.

```
flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

After that, no issues.

```
mattprice&#35;matt-pc ~ $ flatpak install flathub org.gabmus.hydrapaper
Looking for matches…
Required runtime for org.gabmus.hydrapaper/x86_64/stable (runtime/org.gnome.Platform/x86_64/41) found in remote flathub
Do you want to install it? [Y/n]: y

org.gabmus.hydrapaper permissions:
    ipc     fallback-x11     wayland     x11     dri     file access [1]     dbus access [2]     bus ownership [3]

    [1] /usr/share/backgrounds, /usr/share/gnome:ro, home:ro, xdg-config/autostart, xdg-run/dconf, ~/.config/dconf:ro, ~/.config/sway/config, ~/.swaylock/config
    [2] ca.desrt.dconf, org.freedesktop.Flatpak, org.gnome.Mutter.DisplayConfig, org.gnome.SettingsDaemon
    [3] org.gabmus.hydrapaper, org.gabmus.hydrapaper.Daemon


        ID                                             Branch            Op           Remote            Download
 1. [✓] org.freedesktop.Platform.GL.default            21.08             i            flathub           130.9 MB / 131.2 MB
 2. [✓] org.freedesktop.Platform.openh264              2.0               i            flathub             1.5 MB / 1.5 MB
 3. [✓] org.gabmus.hydrapaper.Locale                   stable            i            flathub             3.6 kB / 67.4 kB
 4. [✓] org.gnome.Platform.Locale                      41                i            flathub            17.7 kB / 336.5 MB
 5. [✓] org.gtk.Gtk3theme.Breeze                       3.22              i            flathub           114.0 kB / 162.8 kB
 6. [✓] org.gnome.Platform                             41                i            flathub           241.0 MB / 289.1 MB
 7. [✓] org.gabmus.hydrapaper                          stable            i            flathub             1.1 MB / 1.5 MB

Installation complete.

```

Now, hydrapaper didn't end up working properly, but that's not related to the install process. That app is just broken.  Apparently, it doesn't play well via flatpak install. I Installed VSCode with flatpak and everything is fine otherwise.

EDIT: Hydrapaper is not compatible with Plasma. The "broken" thought was referring to issues people were having on gnome based desktops.

**Comment by TraceyC77** [comment]
This needs to be tested with the 4.6 ISOs to see how things currently stand

**Comment by joebonrichie** [comment]
Resolved #3430
